src/App.tsx

```diff
@@ -1,35 +1,35 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import './App.css';
+import CarForm from './components/car/car_form';
+import { useState } from 'react';
+import CarList from './components/car/car_list';
+
+export interface Car {
+  id: number;
+  maker: string;
+  model: string;
+  year: number;
+}
```

Comment:
You can use here type that you declared in the `types.ts` instead of copying the code.